lib/model.js

```diff
@@ -147,19 +151,32 @@ internals.patchSchema = (schema) => {
     return schema.options({ noDefaults: true });
 };
 
+// Converts a Joi error object to the format the Object.ValidationError constructor expects as input
+// https://github.com/Vincit/objection.js/blob/aa3f1a0bb830211e478aa6a664561155c98850f4/lib/model/ValidationError.js#L10
 internals.parseJoiValidationError = (validation) => {
 
     const errors = validation.error.details;
     const validationInfo = {};
 
+    if (errors.length === 1) {
+        const error = errors[0];
+        validationInfo.message = error.message;
+        validationInfo.data = error.context;
+        validationInfo.type = error.type;
```

Comment:
Is it possible that `type` should be `ValidationErrorType.ModelValidation`?  If so we might need to find a new place to put the Joi error type.

Looking at https://github.com/Vincit/objection.js/blob/aa3f1a0bb830211e478aa6a664561155c98850f4/lib/model/AjvValidator.js#L179

Reply:
ay! huh, I had no idea about any of that business. god, objection is really cool. and huge. man, kinda in awe they created something like that. ANYWAY. thanks for that point, will fix 🙏